microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -229,6 +229,11 @@ For more info about the `DebuggerAddress` property of the `EdgeOptions` object,
 
 Congratulations!  You've successfully automated a WebView2 project and driven WebView2 by using Selenium and Microsoft Edge WebDriver, by attaching Microsoft Edge WebDriver to a running WebView2 app.
 
+### Note for using UWP projects and debugging
+Because of the way that UWP projects are launched the WebDriver is not able to automatically configure the debug settings on launch. You will need to use the registry and enable debugging for all WebView2 process on startup.
```

Comment:
@DavidShoe - "You will need to use the registry" - what about the environment variable approach, is it supported for this scenario or not?  If the environment variable approach is supported, this opening sentence needs to mention "the foo environment variable".  What's the name /spelling of the equivalent environment variable?

Reply:
Only registry.  
HKEY_CURRENT_USER\Software\Policies\Microsoft\edge\WebView2\AdditionalBrowserArguments
with the assigned value in the "*" REG_SZ.  As per our docs.

Reply:
caution, this is an Outdated comment, which means the surrounding code is outdated.

I entered a [fresh comment](https://github.com/MicrosoftDocs/edge-developer/pull/2228/files#r990449408)

Reply:
I don't have a Resolve Comment button.  You can click it.